### PR TITLE
Fix date format

### PR DESCRIPTION
**Pull request by stephanesan**
Incorporate both commits from SiMet and pushed the pattern into the oic types file for reuse across other schema's.

### CODE REVIEW

**Review by craigpratt — APPROVED** [review]
Looks good. Just watch for the description of mndt oic.wk.p-schema.json, since that has a "ReadOnly" added in the description.

File: schemas/oic.wk.p-schema.json [hunk 1]
```diff
@@ -30,11 +30,9 @@
           "description": "ReadOnly, Model number as designated by manufacturer"
         },
         "mndt": {
-          "type": "string",
+          "$ref": "oic.types-schema.json#/definitions/date",
           "readOnly": true,
-          "description": "ReadOnly, Manufacturing Date as defined in ISO 8601, where the format is [yyyy]-[mm]-[dd].",
-          "pattern": "^([0-9]{4})-(1[0-2]|0[1-9])-(3[0-1]|2[0-9]|1[0-9]|0[1-9])$",
-          "format": "date-time"
+          "description": "ReadOnly, Manufacturing Date."
```
Comment (craigpratt) [inline schemas/oic.wk.p-schema.json]: Of course, "ReadOnly" should not be added in the description.

Comment (stephanesan) [reply schemas/oic.wk.p-schema.json]: It should and is not added (it is left untouched here). Removing it (consistently over all schema's) though is part of another pull request https://github.com/openconnectivityfoundation/core/pull/14.
But you are correct in that care will have to be taken that this patch doesn't conflict and overwrite the other patch ( by reintroduces the readonly description).

**Comment by SiMet** [comment]
@stephanesan for CTT we'd like to have format property in schema. However, in oic.wk.p there was "date-time" but should be "date"

**Comment by stephanesan** [comment]
Yes, I am trying to track back what happened :-(

File: schemas/oic.wk.p-schema.json (hunk 1 above)
Comment (stephanesan) [reply schemas/oic.wk.p-schema.json]: I resolved the conflict with the latest master so the merge can happen smoothly now.

**Pull request merged**